tests/k8s/helpers.bash

```diff
@@ -5,6 +5,8 @@ source "${dir}/../helpers.bash"
 # dir might have been overwritten by helpers.bash
 dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
+MAX_LOG_PRINT=30
```

Comment:
Thanks for adding this!

Reply:
Ah, I'll actually drop this change out from this series because it's unrelated, but I do plan on following up on this separately.